Project.toml

```diff
@@ -4,17 +4,17 @@ authors = ["Pietro Vertechi"]
 version = "0.3.2"
 
 [deps]
-BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
+CodecZlib = "944b1d66-785c-5afd-91f1-9de20f533193"
```

Comment:
The new dependencies also need compatibility information (the `[compat]` field below).